<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtUserName" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>80</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>60</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>User Name:</string>
   </property>
   <property name="buddy" >
    <cstring>txtUserName</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPassword" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>150</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
   <property name="echoMode" >
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="Password" stdset="0" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>130</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Password:</string>
   </property>
   <property name="buddy" >
    <cstring>txtPassword</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>OPIM REGISTRATION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegister" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>210</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text" >
    <string>Register</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
